dashboard/runs/cancel-run.md

```
@@ -81,39 +96,36 @@ The associated Currents run will be cancelled with the corresponding notes:
 
 <figure><img src="../../.gitbook/assets/currents-2023-07-04-14.18.17@2x.png" alt=""><figcaption></figcaption></figure>
 
-#### Cancelling with CI Build ID
+#### Cancelling with an API key
```

markdownlint-cli2 (0.23.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)